src/main/scala/loamstream/drm/lsf/BacctAccountingClient.scala

```diff
@@ -50,9 +50,22 @@ final class BacctAccountingClient(
      * Documentation on bacct's output format:
      * https://www.ibm.com/support/knowledgecenter/en/SSWRJV_10.1.0/lsf_command_ref/bacct.1.html
      */
-    val node = rawBacctOutput.collectFirst { case Regexes.node(n) => n }
+    def stripLineEndings(s: String): String = s.replaceAll("\\n", "").replaceAll("\\r", "")
     
-    val queue = rawBacctOutput.collectFirst { case Regexes.queue(q) => q }.map(Queue(_))
+    val delim = "%%%%%%%%%%%%"
+    
+    val joinedBacctOutput = rawBacctOutput.mkString(delim)
+    
+    def munge(s: String): String = s.replaceAll(s"${delim}\\s*", "")
+    
+    def extract(regex: Regex): Option[String] = joinedBacctOutput match {
+      case regex(s) => Option(stripLineEndings(munge(s).trim))
+      case _ => None
+    }
```

Comment:
I'll see.  Line-ending characters and any whitespace immediately following them need to be removed, since `bacct` adds lots of line breaks and indentation.  (See previous comment.)  The shelling-out-to-bacct code returns stdout and stderr as sequences of lines (that was/maybe-is more convenient) and not single strings with embedded line-ending chars.  

Your code traverses those sequences, but the lines themselves need to be joined first before there are any newlines to remove.  Bacct potentially splits the values this code needs to parse over multiple lines (see previous comment).  I admit this situation is a bit of a mess.